ts/Dashboards/Actions/Bindings.ts

```diff
@@ -62,62 +62,34 @@ class Bindings {
     public static addComponent(
         options: Bindings.ComponentOptions,
         cell?: Cell
-    ): ComponentTypes | undefined {
+    ): Component | undefined {
         const componentContainer = document.getElementById(options.cell);
         const optionsStates = options.states;
         const optionsEvents = options.events;
 
         cell = cell || Bindings.getCell(options.cell);
-        let component: ComponentTypes|undefined;
+        let component: Component|undefined;
```

Comment:
I guess that there should be also extra space around the separator `|` like in other parts of the code.